x-pack/test/security_solution_endpoint/config.ts

```diff
@@ -16,6 +20,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     ...xpackFunctionalConfig.getAll(),
     pageObjects,
     testFiles: [resolve(__dirname, './apps/endpoint')],
+    dockerServers: createEndpointDockerConfig(),
```

Comment:
Will this cause an error for anyone that attempt to run the tests locally, but does not have docker installed?

I was thinking that the `cockerServers` setup would only be added to the config if we detect that we will be running with it (aka: not against the public registry). 

(I don't currently have docker installed, so I will check this out next and try to run it)

Reply:
Nope it shouldn't cause any errors! Adding the config doesn't affect anything unless the port is set. That'll only be set if the environment variable is set.

So everything should be like before where you can run the tests using the public package registry.

Reply:
👍